server/routes.js

```diff
@@ -54,7 +54,7 @@ module.exports = (app) => {
   app.get('/:slug/:tier/banner.md', mw.cache(300), mw.fetchGroupBySlug, mw.fetchUsers(), banner.markdown);
   app.get('/:slug/:tier/badge.svg', mw.cache(300), mw.fetchUsers(), banner.badge);
   app.get('/:slug/badge/:tier.svg', mw.cache(300), mw.fetchUsers(), banner.badge);
-  app.get('/:slug/:tier/:position/avatar(.png)?(.jpg)?', mw.cache(300), mw.ga, mw.fetchUsers({cache: 300}), banner.avatar);
+  app.get('/:slug/:tier/:position/avatar(.:format)?', mw.cache(300), mw.ga, mw.fetchUsers({cache: 300}), banner.avatar);
```

Comment:
Does this yield the intended behaviour if e.g calling `GET /:slug/:tier/:position/avatar.blah`? Route `.../avatar\.(png|jpg|svg)` [should work](http://forbeslindesay.github.io/express-route-tester/). And according to [the doc](http://expressjs.com/en/api.html#req.params) format can be fetched with something like `req.params[0]` if you create a capture group (I guess with parentheses like `.../avatar\.\((png|jpg|svg)\)`).